mlflow/store/cli.py

```diff
@@ -0,0 +1,101 @@
+from mlflow.utils.logging_utils import eprint
+
+import click
+
+from mlflow.tracking import _get_store
+from mlflow.store.artifact_repo import ArtifactRepository
+from mlflow.utils.proto_json_utils import message_to_json
+
+
+@click.group("artifacts")
+def commands():
+    """Upload, list, and download artifacts from an MLflow artifact repository."""
+    pass
+
+
+@commands.command("log-artifact",
+                  help="Logs a local file as an artifact of a run, optionally within a " +
+                       "run-specific artifact path. Run artifacts can be organized into " +
+                       "directories, so you can place the artifact in a directory this way.")
+@click.option("--local-file", "-l", required=True,
+              help="Local path to artifact to log")
+@click.option("--run-id", "-r", required=True,
+              help="Run ID into which we should log the artifact.")
+@click.option("--artifact-path", "-a",
+              help="If specified, we will log the artifact into this subdirectory of the " +
+                   "run's artifact directory.")
+def log_artifact(local_file, run_id, artifact_path):
+    """
+    Creates a new experiment in the configured tracking server.
+    """
+    store = _get_store()
+    artifact_uri = store.get_run(run_id).info.artifact_uri
+    artifact_repo = ArtifactRepository.from_artifact_uri(artifact_uri, store)
+    artifact_repo.log_artifact(local_file, artifact_path)
+    eprint("Logged artifact from local file %s to artifact_path=%s" % (local_file, artifact_path))
+
+
+@commands.command("log-artifacts",
+                  help="Logs the files within a local directory as an artifact " +
+                       "of a run, optionally within a run-specific artifact path. Run " +
+                       "artifacts can be organized into directories, so you can place the " +
+                       "artifact in a directory this way.")
+@click.option("--local-dir", "-l", required=True,
```

Comment:
Ditto on name

Reply:
Hmm, this one actually can't be a file in GC, DBFS, or S3 either.